src/tests/data_type/decimal.rs

```diff
@@ -0,0 +1,26 @@
+use {crate::*, executor::Payload, prelude::Value::*};
+
+test_case!(decimal, async move {
+    let test_cases = vec![
+        (
+            "CREATE TABLE DECIMAL_ITEM (decimal_field DECIMAL)",
+            Ok(Payload::Create),
+        ),
+        (
+            r#"INSERT INTO DECIMAL_ITEM VALUES (1)"#,
+            Ok(Payload::Insert(1)),
+        ),
```

Comment:
Current integration test is enough for the system.
However, this integration test would be also a good material for human, it is kind of a usage guide.
`DECIMAL`'s specialty compared to `FLOAT` is that we can specify numbers in exact precision.

Let's just have an example to insert `23.18273` like non-integer number.
You can declare decimal customly by
e.g.
```rust
let decimal = |num, scale| rust_decimal::Decimal::new(num, scale);
```